l10n_be_account_bank_statement_import_bpost/account_bank_statement_import_bpost.py

```diff
@@ -0,0 +1,97 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    l10n_be_account_bank_statement_import_bpost module for Odoo
+#    Copyright (C) 2015 Akretion (http://www.akretion.com)
+#    @author Alexis de Lattre <alexis.delattre@akretion.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import logging
+from datetime import datetime
+from StringIO import StringIO
+from openerp import models, fields, _
+import unicodecsv
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountBankStatementImport(models.TransientModel):
+    _inherit = 'account.bank.statement.import'
+
+    def _check_bpost(self, cr, uid, data_file, context=None):
+        return data_file[3:].startswith(
+            u'"Numéro de compte :";"'.encode('utf-8'))
```

Comment:
This code will need to be adapted if the file is in Dutch. But I don't have an example of a file in Dutch, because my customer is French.